INSTALL.md

```diff
@@ -1,3 +1,85 @@
 # Installation
 
 See https://github.com/nerdvegas/rez/wiki/Getting-Started#installation
+
+
+First, install Rez. Download the source, and, from the source directory, run:
```

Comment:
Philosophical question (sorry in advance): Should we move this doc in the https://github.com/nerdvegas/rez/wiki/Getting-Started#installation doc? I'm also wondering if we should add a mention about how to install in the README as it's usually a common practice to do so.

Or we want to tackle that into a separate subject and wait for your roadmap before deciding?

Reply:
Yes to both, will make those changes.